packages/components/src/progress_bar/index.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { getPercentage } from './utils';
+import styles from './index.module.scss';
+
+const ProgressBar = ({
+  state,
+  size,
+  children,
+  progressBarStyles,
```

Comment:
`className` maybe?

Reply:
Because I think `className` and `progressIndicatorClassName` wouldn't really reflect it since `className` is very generic. Nevertheless I agree, my names are pretty bad

Reply:
Why do we need those className props anyway? Any chance you could provide links / screenshots?

Reply:
To provide a way to style the 2 progress lines. In this case lightblue and darkblue with border-radius.  

![Uploading Screenshot 2022-05-24 at 17.08.16.png…]()

If nothing is passed, the default looks are applied, but if I wouldn't pass them, I style these child elements in a clean way as far as I'm concerned. 
Hope that clears it up.

Reply:
https://github.com/goodhood-eu/goodhood/pull/158#issuecomment-1135985331 @ozerovs  is very correct about me not liking to pass too many classNames :D 

maybe introduce a `theme` prop? 

you could predefine two themes:
- legacy (legacy.module.scss)
- modern (modern.module.scss)

Then it should be a lot easier to refactor in the future. With passing in class names you never know what will break if you change one single css prop